src/mumble/Log.ui

```diff
@@ -193,7 +198,27 @@
         </property>
        </widget>
       </item>
-     </layout>
+	  <item>
+        <widget class="QCheckBox" name="qcbMessageThreshold">
+         <property name="toolTip">
+          <string>If checked, notifications will not be played when more than 20 people are active on the server.</string>
+         </property>
+         <property name="text">
+          <string>Don't play notifications when there are more than the following number of users on the server.</string>
+         </property>
+		</widget>
+       </item>
+       <item>
+	    <widget class="QSpinBox" name="qsbMessageThresholdUsers">
+	     <property name="toolTip">
+		  <string>Number of users that will trigger thresholding functionality.</string>
+		  </property>
+		  <property name="frame">
+		   <bool>true</bool>
+		  </property>
+		 </widget>
```

Comment:
This is spaces vs. tabs, huh? I need to change my visual studio settings to fix it I guess.

Reply:
Are you editing these UI files manually or via Qt Designer?

Reply:
I did these manually...

:stuck_out_tongue_winking_eye: